tests/scripts/lib/docker.py

```diff
@@ -96,8 +96,8 @@ class DockerRocky8Container(DockerCentosContainer):
 class DockerDebianContainer(DockerContainer):
 
     def start_sshd(self):
-        self.exec('/bin/chmod 0755 /run/sshd')
-        self.exec('/bin/systemctl start ssh')
+        # self.exec('/bin/chmod 0755 /var/run/ssh') location doesn't exist in debian10 testing
```

Comment:
This change is effective for both debian9 and debian10. Could you please confirm this change does not affect the execution of the tests on debian9 too?